sparkles/core.py

```diff
@@ -170,7 +170,13 @@ def _run_aca_review(load_name=None, *, acars=None, make_html=True, report_dir=No
         aca.check_catalog()
 
         if roll_level == 'all' or aca.messages >= roll_level:
-            aca.get_roll_options()  # sets roll_options, roll_info attributes
+            try:
+                aca.get_roll_options()  # sets roll_options, roll_info attributes
+            except Exception: # as err:
+                err = traceback.format_exc()
+                aca.add_message('critical', text=f'Running get_roll_options() failed: \n{err}')
+                aca.roll_options = None
```

Comment:
This looks great to me but I'm not clear on if it is an API change (were these None when this failed previously or did the attributes just not exist?).

Reply:
The `None` values correspond to the defaults in `__init__`, so these are consistent with no attempt to get roll options.  So the API is to rely on messages to see a failure.  Hopefully this never actually gets used, but I think that seeing a critical message at this level will get the required attention.

Reply:
@jskrist - this has the potential to put in a multiline message with a `\n` in it.  It works fine in HTML.  I assume within Windows MATLAB that this won't *break* anything, and at worst just make something that isn't formatted perfectly.  I could probably make that `\n` into `os.linesep()`, but I don't know if that will help out.

BTW the only way I can think of to test this in ORviewer is to hack the code (put in a `x = 1/0` statement) and install that.   Right now I don't know any way to break the code from the outside. 

One way which I plan for testing is to make an `acar` object and then do `del acar.stars['ra']` and then do the review with roll options.   But I don't think that helps in ORviewer integration testing.

Reply:
@taldcroft - thanks for the heads up.  I can force the situation in ORViewer in a similar way (run a schedule update and check for stars [with a breakpoint between where it creates the ACAReviewTables and where it runs the report with roll_options], then manually modify the acar object and then do the review with roll options.  It is not easy and won't be able to be easily automated, but I can do it manually for now.

Also, I _think_ the newline character ("\n") will be interpreted correctly, but I will double check, as I might be escaping any backslashes that I don't expect (for safety).